src/main/java/net/minestom/server/network/socket/Worker.java

```diff
@@ -19,11 +19,15 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.atomic.AtomicInteger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @ApiStatus.Internal
 public final class Worker extends MinestomThread {
     private static final AtomicInteger COUNTER = new AtomicInteger();
 
+    static final Logger LOGGER = LoggerFactory.getLogger(Server.class);
```

Comment:
Referencing another classes logger seems very strange, but will be fixed by adding a close method here

Reply:
As this had already been done before in other forms, I thought it wouldn't be a problem. A little hint, there are classes that continue to do this

Reply:
No need to add another case :)